sdk/core/core-https/CHANGELOG.md

```diff
@@ -3,6 +3,7 @@
 ## 1.0.0-beta.2 (Unreleased)
 
 - Changed from exposing `DefaultHttpsClient` as a class directly, to providing `createDefaultHttpsClient()` to instantiate the appropriate runtime class.
+- Fix an issue when passing in proxy hosts. There's some difference between the previous `tunnel.HttpsOverHttpsOptions` and `HttpsProxyAgentOptions`, in particular the former's `host` property includes both the protocol and the host parts, while the later have separate `host` and `protocol` properties.
```

Comment:
I think this is maybe a little too detailed (and nobody reading these notes cares much about tunnel.) 

Really we're just fixing a bug where we're calling a transport layer incorrectly, and I don't think consumers need to do anything differently except continue to set the proxy variables as they expected to?